src/views/Auth/Login.jsx

```diff
@@ -21,6 +22,10 @@ export default function Login() {
 
     // TODO: If login was unsuccessful, set an error with a message
     // to display to the user that their login failed.
+    !loginWasSuccessful
+      ? alert('you username and password dont match')
```

Comment:
The codebase was set up with an `error` state that would display a message without an alert